### DISCUSSION TITLE
Non-scalar datatypes in NoSQL databases

### DISCUSSION THREAD

**Discussion by felipefrancesco-yapoli** [owen-it/laravel-auditing#789] [Q&A]
| Q                 | A
| ----------------- | ---
| Bug?              | no
| New Feature?      | yes
| Framework         | Laravel
| Framework version | 5.8.0
| Package version   | 10.0
| PHP version       | 7.4.4

Hi, I've been trying to use Laravel Auditing with a DynamoDB database. Functionally it works fine and is able to record every CRUD events on models. However I do have some fields which are non-scalar types which in the model are mapped as arrays. So, I realized that non-scalar types are skipped when recording old and new values.

Is there anyone using this kind of NoSQL database and/or is there any new feature planned to be able to record those non-scalar types?

Thanks!

**Comment by infinitypaul** [comment]
@felipefrancesco-yapoli  try this out if it helps https://github.com/infinitypaul/laravel-dynamodb-auditing it integrate well with dynamodb
